ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -62,11 +72,29 @@ export class ImportCommand extends IronfishCommand {
       account.version = ACCOUNT_SCHEMA_VERSION as number
     }
 
-    const result = await client.importAccount({
-      account,
-      rescan: flags.rescan,
-    })
-
+    let result: RpcResponseEnded<{ name: string; isDefaultAccount: boolean }>
+    if ('spendingKey' in account && account.spendingKey) {
+      result = await client.importSpendAccount({
+        account: {
+          name: account.name,
+          spendingKey: account.spendingKey,
+          version: account.version,
+        },
+        rescan: flags.rescan,
+      })
+    } else {
+      result = await client.importViewAccount({
+        account: {
+          name: account.name,
+          viewKey: account.viewKey ?? '',
+          publicAddress: account.publicAddress ?? '',
+          incomingViewKey: account.incomingViewKey ?? '',
+          outgoingViewKey: account.outgoingViewKey ?? '',
+          version: account.version,
+        },
+        rescan: flags.rescan,
+      })
+    }
```

Comment:
If you split `AccountImport` into two types, this should reduce to something like follows:
```suggestion
      const rescan = flags.rescan
      result = 'spendingKey' in account  ? 
      await client.importSpendAccount({account, rescan}) : 
      await client.importViewAccount({account, rescan})
```
syntax probably isn't right for awaits, but you get the point